File: Python/problem006.py
#! /usr/bin/env python3
'''
Problem 6 - Project Euler
http://projecteuler.net/index.php?section=problems&id=6
'''
if __name__ == '__main__':
    print(sum(range(1,101))**2 - sum([x**2 for x in range(1,101)]))

File: Clojure/problem006.clj
#! /usr/bin/env clojure
; Problem 6 - Project Euler
; http://projecteuler.net/index.php?section=problems&id=006

(-> (range 1 (inc 100))
    (as-> x
          (let [sum (reduce + x)]
            (- (* sum sum) (reduce + (map #(* % %) x)))))
    (println))
